CHANGELOG.md

```diff
@@ -98,6 +98,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - The "Node Output" page has been relabelled as "Troubleshooting" in the documentation and CLI commands for troubleshooting have been added to it.
 
+### Changed
+
+- The endpoint `GET /node/js_metrics` now also returns the QuickJS runtime maximum execution time
```

Comment:
This should be highlighted under `Operations` as a new feature. We should say both that there _are_ JS execution caps (for memory sizes and execution time) that can be controlled by governance, and that the current caps can be fetched from `/js_metrics`.

Reply:
I have pushed this up and under `Operations`. I also reworded to highlight that all three options can be changed/fetched although the previous PR talks about memory options as well. Happy to have the CHANGELOG `suggestion` to alter it if someone feels otherwise.